filters/auth/forwardtokenpart.go

```diff
@@ -0,0 +1,74 @@
+package auth
+
+import (
+	"fmt"
+
+	"github.com/zalando/skipper/filters"
+	"golang.org/x/net/http/httpguts"
+)
+
+const (
+	ForwardTokenPartName = "forwardTokenPart"
```

Comment:
I know naming is hard but maybe it should be `forwardTokenField` as `part` feels a bit ambiguous?..
Also do we foresee the need to forward multiple fields e.g. comma-separated?